tidb-cloud/ticloud-config-set.md

```diff
@@ -0,0 +1,65 @@
+---
+title: ticloud config set
+Summary: The reference of `ticloud config set`.
+---
+
+# ticloud config set
+
+You can use `ticloud config set` to configure the properties for the active [user profile](/tidb-cloud/cli-reference.md#user-profile) as follows:
```

Comment:
```suggestion
Configure the properties for the active [user profile](/tidb-cloud/cli-reference.md#user-profile):
```

Reply:
Change reason: To keep consistent style with the short description of other commands.